cmake/modules/FindPortAudio.cmake

```diff
@@ -78,5 +83,8 @@ if(PortAudio_FOUND)
         INTERFACE_COMPILE_OPTIONS "${PC_PortAudio_CFLAGS_OTHER}"
         INTERFACE_INCLUDE_DIRECTORIES "${PortAudio_INCLUDE_DIR}"
     )
+    if(PortAudio_ALSA_H)
+      target_compile_definitions(PortAudio::PortAudio INTERFACE PA_USE_ALSA)
+    endif()
```

Comment:
This should be handled by PortAudio's build system. I opened a PR upstream to implement that: https://github.com/PortAudio/portaudio/pull/635